pkgs/development/compilers/llvm/10/default.nix

```diff
@@ -30,16 +30,11 @@ let
   in {
 
     llvm = callPackage ./llvm.nix { };
```

Comment:
Why not enable polly here? It would merely add a new shared library. Then the clang build would pick up the polly functionality and make it available though a flag.

Reply:
Maybe I am deeply confused, but do you expect that `polly` is now available in the vanilla `clang`?

Reply:
I totally managed to not add llvm.nix . Sorry, I enable it by default.